src/test/resources/pages/instructorCourseEnrollError.html

```diff
@@ -20,32 +20,13 @@ <h1>
         </strong>
```

Comment:
What is "relevant HTML files"? What is this commit doing?

Reply:
Explained in https://github.com/TEAMMATES/teammates/pull/8884#discussion_r200568395